src/main/java/life/qbic/data/processing/AppConfig.java

```diff
@@ -36,15 +36,18 @@ ScannerConfiguration scannerConfiguration(
   RegistrationWorkersConfig registrationWorkersConfig(
       @Value("${registration.threads}") int amountOfWorkers,
       @Value("${registration.working.dir}") String workingDirectory,
-      @Value("${registration.target.dir}") String targetDirectory) {
-    return new RegistrationWorkersConfig(amountOfWorkers, workingDirectory, targetDirectory);
+      @Value("${registration.target.dir}") String targetDirectory,
+      @Value("${registration.metadata.filename}") String metadataFileName) {
+    return new RegistrationWorkersConfig(amountOfWorkers, workingDirectory, targetDirectory,
+        metadataFileName);
   }
 
   @Bean
   RegistrationConfiguration registrationConfiguration(
       RegistrationWorkersConfig registrationWorkersConfig) {
     return new RegistrationConfiguration(registrationWorkersConfig.workingDirectory().toString(),
-        registrationWorkersConfig.targetDirectory().toString());
+        registrationWorkersConfig.targetDirectory().toString(),
```

Comment:
Question: Should we check if the Path is valid (aka the File/Folder exists) during config initialization?